client/blocks/eligibility-warnings/style.scss

```diff
@@ -12,6 +14,14 @@
 		display: flex;
 		gap: 5px;
 	}
+
+	.card {
+		box-shadow: none;
```

Comment:
Can we limit this to the modal? Install plugin screen looks kind of borderless:

![Screenshot 2022-09-29 at 17-49-22 Plugins ‹ Bussiness Simple — WordPress com](https://user-images.githubusercontent.com/811776/192972537-f5aaf22a-4557-4b92-91ad-32992c01ac1b.png)

Reply:
Good suggestion @lsl, I added a default box-shadow to the main  `eligibility-warnings` box and removed it when inside  `.dialog` box:

![image](https://user-images.githubusercontent.com/1044309/193023799-c2475ae1-35d5-457d-b7f7-cb606d52140d.png)